index.html

```diff
@@ -230,5 +230,46 @@ <h2>skills</h2>
         </li>
       </ul>
     </section>
+    <section class="contact-form-section">
+      <h3>Contact me</h3>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form
+        action="https://formspree.io/f/{form_id}"
+        method="post"
+        class="form-section"
+      >
+        <label for="full-name">
+          <input
```

Comment:
- [ ] Good job implementing the contact form with the `formspree` server, 👏 . The response of the server should be successful , however I noticed that the response of the `formspree` server is not so 😄 , Kindly make sure you fix that to meet the requirement 👍 

![Capture](https://user-images.githubusercontent.com/80458199/235794544-08d53829-17d5-4bd7-9a6d-8f7d765af359.PNG)